CHANGELOG.md

```diff
@@ -1,21 +1,32 @@
 # Changelog
+
 All notable changes to this project will be documented in this file. The format (starting with 18.0.0) is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 This project does NOT adhere to [Semantic Versioning](https://semver.org/spec/v2.0.0.html), and major versions of this project denote compatibility with Sitecore Platform versions. Refer to the "Headless Services" section in the [Sitecore modules compatibility table](https://support.sitecore.com/kb?id=kb_article_view&sysparm_article=KB0541788) or the [Headless Rendering download page](https://dev.sitecore.net/Downloads/Sitecore_Headless_Rendering.aspx) for more details on versioning.
 
+## 19.0.3
+
+## Bug Fixes
+
+`[sitecore-jss-react]` `[sitecore-jss-nextjs]`
+
+- [Next.js][react] Link component does not add anchor to the internal links ([#1228](https://github.com/Sitecore/jss/pull/1228))
+
 ## 19.0.2
 
 ## Bug Fixes
 
 `[sitecore-jss-vue]`
-* [EE] Cannot add a new rendering to the newly created ([#902](https://github.com/Sitecore/jss/pull/902))
```

Comment:
Looks like your editor made some other formatting changes automatically. Can you please revert these? Maybe stick with the "*" bullet (what we've used thus far) for the new content too.